src/applications/mhv/landing-page/utilities/api/getApiPath.jsx

```diff
@@ -0,0 +1,19 @@
+import environment from '~/platform/utilities/environment';
+import { apiRequest } from '~/platform/utilities/api';
+
+const apiBasePath = `${environment.API_URL}/my_health/v1`;
+
+/**
+ * Gets the folder list for the current user.
+ * @returns
+ */
+export const getFolderList = () => {
+  return apiRequest(
+    `${apiBasePath}/messaging/folders?page=1&per_page=999&useCache=false`,
+    {
+      headers: {
+        'Content-Type': 'application/json',
+      },
+    },
```

Comment:
I think the `page` and `perPage` query params are unnecessary, here. Specifying `headers` isn't needed since the api is configured to respond using json.

Reply:
re: `page` and `perPage` query params.

i have no issue getting rid of them, but the original author of the SM dot work included them i assume for good reason. are there docs for the .../messaging/folders endpoint that suggest these are optional params (and we can dispense with them)?